### ISSUE TITLE
Trait calls for Tuple getters don't work.

### ISSUE THREAD

**Issue by gusty** [fable-compiler/Fable#1991]
### Description

Trait calls for Tuple getters don't work.

### Repro code

```
// tried with both alternative syntax
type Item1 = static member inline Invoke value = (^t : (member get_Item1 : unit ->_) value)
type Item2 = static member inline Invoke value = (^t : (member Item2 : _) value)

let a = Item1.Invoke (1,2,3)
let b = Item2.Invoke (1,2,3)
``` 

### Expected and actual results

Expected (from fsi)

```
type Item1 =
  class
    static member
      Invoke : value: ^t -> 'a when  ^t : (member get_Item1 :  ^t -> 'a)
  end
type Item2 =
  class
    static member
      Invoke : value: ^t -> 'a when  ^t : (member get_Item2 :  ^t -> 'a)
  end
val a : int = 1
val b : int = 2
```

Actual

```
FABLE: Cannot resolve trait call get_Item1 - Inline call from .(4,8)
(1,50)
FABLE: Cannot resolve trait call get_Item2 - Inline call from .(5,8)
(2,50)
```

### Related information

* Fable version: I tried it with fable-splitter 2.1.12 and fable-compiler 2.4.18 but it also fails in the online REPL.
* Operating system: Windows in my case, but don't think it's relevant.

**Comment by MangelMaxime** [comment]
Thank you @ncave for the fix.

@gusty The fix has been released in version 2.4.19

**Comment by gusty** [comment]
Thanks for the quick resolution !
